deploy/centos7x64-src.sh

```diff
@@ -7,15 +7,15 @@ yum -y install smartmontools
 yum -y install epel-release
 yum -y install certbot
 cd /usr/src
-wget https://nodejs.org/dist/v12.0.0/node-v12.0.0.tar.gz
-tar zxf node-v12.0.0.tar.gz
-rm -f ./node-v12.0.0.tar.gz
-cd node-v12.0.0
+wget https://nodejs.org/dist/v12.4.0/node-v12.4.0.tar.gz
```

Comment:
I'd suggest to put a variable with `NODE_VERSION='v12.4.0'` at the top of such files and then use it as 
```bash
wget "https://nodejs.org/dist/${NODE_VERSION}/node-${NODE_VERSION}.tar.gz"
``` 
and so on to avoid changing all this boilerplate.

Reply:
But we will still change this variable in all files, maybe there are url to the latest ver?